tox.ini

```diff
@@ -1,24 +1,7 @@
 [tox]
-envlist =
-    py36,py34,py27
+envlist = py{27,34,35,36}
 skip_missing_interpreters = true
 
 [testenv]
-commands =
-    python -m pytest {posargs}
-deps =
-    pytest
-    mock
-    coverage
-    pytest-clld
-    pytest-cov
-    pytest-mock
-
-[testenv:py36]
-basepython = python3.6
-
-[testenv:py34]
-basepython = python3.4
-
-[testenv:py27]
-basepython = python2.7
+extras = test
+commands = pytest {posargs}
```

Comment:
AFAICT the selenium tests do not work when run with tox. Thus, what I ended up doing for Glottolog is marking the selenium tests using a decorator
```python
@pytest.mark.selenium
def test_site_search(selenium):
    ...
```
and changing the command in tox.ini to
```
commands =
    python -m pytest -m "not selenium" {posargs}
```